mongodb-developer/leafsteroids
# Use extended reference pattern for Player and Event.

Adjust the REST service to the ext. ref. pattern as shown in `extended_ref.js`.

How to test:

Edit Local `.env` file and set appropriate CONNECTION_STRING and DATABASE_NAME

Run and open:
http://localhost:8000/swagger/index.html

Try and execute: `POST /Recordings`
Change the event id to `mdb-internal` (an incorrect event id will lead to an error).
Post the entry.
Check the `Leafsteroids_stage/recordings` for the new entry with the expected pattern.
